pwa-devdocs/src/pagebuilder/index.md

```diff
@@ -1,5 +1,41 @@
 ---
-title: Page Builder
+title: Page Builder integration
 ---
 
-Summary paragraph goes here.
+The purpose of the Page Builder to PWA integration is to retrieve data from the Page Builder content types and push that data into equivalent React components for display in a PWA app. The Page Builder framework populates the React components with the properties extracted from the master format HTML, then renders those components to the PWA Studio's Venia app. A visualization of this process follows:
+
+![Page Builder Integration Overview](PageBuilderIntegration.svg)
+
+## Page Builder PWA framework
+
+As part of the Page Builder to PWA integration, we implemented a framework that converts Page Builder’s master format (HTML) into a structured format that works in React and PWA Studio. The key parts of the framework are described as follows:
+
+**RichContent**: The `<RichContent />` component provides the entry point into the Page Builder PWA framework. It controls whether the HTML passed by the PWA app contains Page Builder content. If it does, the HTML is sent to the `<PageBuilder />` component for processing. If not, the HTML is sent directly to the PWA app for display.
+
+**PageBuilder**: The `<PageBuilder />` component directs the parsing of the master format HTML as well as retrieving, populating, and returning content type components back to the Venia app.
+
+**Master Format parser**: The master format parser function (`parseStorageHtml()`) decomposes the master format HTML into the content type HTML fragments (HTMLElements) that compose the master format. The parser sends the content type HTML to the correct property aggregator (`configAggregator`) using the configuration object.
+
+**Configuration object**: The configuration object (`contentTypesConfig`) provides a kind of registry for retrieving a content type's property aggregator and corresponding Page Builder component. The configuration object provides the integration point for your own custom content type components. See [Set up component] for details.
```

Comment:
We actually modified this a little to include a method for retrieving configuration: https://github.com/magento/pwa-studio/blob/feature/page-builder/packages/venia-ui/lib/components/RichContent/PageBuilder/config.js